packages/store/src/search.ts

```diff
@@ -0,0 +1,146 @@
+import { Document as DocumentIndexer, DocumentSearchOptions } from 'flexsearch';
+import { Doc, Map as YMap, Text as YText } from 'yjs';
+import type { YBlock } from './space';
+
+export type QueryContent = string | Partial<DocumentSearchOptions<boolean>>;
+
+function tokenizeZh(text: string) {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  const tokenizer = Intl?.v8BreakIterator;
```

Comment:
Can we replace this non-standard API with standardized [Intl.Segmenter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Segmenter)?

Reply:
v8BreakIterator can support chromium after 2018, Intl.Segmenter add official support after 2020/11, maybe we can make v8BreakIterator as a fallback

Reply:
Browser support after 2020 looks promising. I think it'll be fine to support browser versions within a year for the oversea market, which is what [photopea](https://www.photopea.com/) is doing.